src/components/Camera/CameraOptionsModal.js

```diff
@@ -6,7 +6,8 @@ import { t } from "i18next";
 import { ObsEditContext } from "providers/contexts";
 import * as React from "react";
 import { IconButton } from "react-native-paper";
-import colors from "styles/colors";
+
+import colors from "../../../tailwind-colors";
```

Comment:
Whether we use this file or import directly from `tailwind.config.js`, I think it would help a lot if we used a module alias in `babel.config.js` so we're not constantly juggling these relative paths.